controllers/admin/AdminCartsController.php

```diff
@@ -788,7 +788,18 @@ public function displayAjaxSearchCarts()
 
     public function ajaxReturnVars()
     {
-        $id_cart = (int) $this->context->cart->id;
+        if ((int) Tools::getValue('id_cart')) {
+            $this->context->cookie->id_cart = $id_cart = (int) Tools::getValue('id_cart');
+        } elseif (!$this->context->cookie->id_cart) {
+            $cart = new Cart();
+            $cart->id_currency = $this->context->currency->id;
+            $cart->id_lang = $this->context->language->id;
+            $cart->save();
+            $this->context->cookie->id_cart = $id_cart = (int) $cart->id;
+            $id_cart = (int) $cart->id;
```

Comment:
It seems like you assign `$id_cart`'s value two times here (line 798 and 799)

Reply:
I think I was supposed to remove line 799 indeed! I forgot! I will make the changes.